### PR TITLE
update django-nose

### PR DESCRIPTION
**Pull request by muhammad-ammar**

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@wedaly @macdiesel @symbolist @muzaffaryousaf Please have a look. Thanks

**Comment by macdiesel** [comment]
:+1: 

Can you please add xserver to this page:
https://openedx.atlassian.net/wiki/display/TNL/Library+updates+needed+for+Django+upgrade+to+1.8

**Comment by symbolist** [comment]
:+1:

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Django >= 1.4, < 1.5
 pika
-django_nose
+django_nose=1.4.1
```
Comment (feanil) [inline requirements.txt]: You need a double equal sign.  This is broken.  @muhammad-ammar @symbolist  @wedaly @macdiesel FYI

**Comment by muhammad-ammar** [comment]
@feanil Sorry for this and Thanks a lot..strangely tests were passed with bad syntax.

**Comment by feanil** [comment]
@muhammad-ammar which tests did you run?  there are no tests that automatically run when you make a PR against the XServer.

**Comment by muhammad-ammar** [comment]
Oh...but build was running when I opened the PR..so I thought tests were
running.. shouldn't every repo has automated tests?

**Comment by feanil** [comment]
It should but not all of them do.

@jzoldak @benpatterson FYI

**Comment by benpatterson** [comment]
@muhammad-ammar are you going to fix the equals sign?

**Comment by feanil** [comment]
@benpatterson I already merged a PR that fixed it.  I just wanted to make you aware that this repos is not running any tests on PRs.

**Comment by benpatterson** [comment]
Ok gotcha.

And yeah CI is an important TODO. I'm glad you pointed that out. I'll shoot you a note on this as I have additional questions.

**Pull request merged**